backend/apps/notification/selectors.py

```diff
@@ -0,0 +1,22 @@
+from .models import Notification
+
+
+def get_notifications(user, is_read=None, is_important=None, notification_type=None):
```

Comment:
Rename these to follow the convention:

notification_get, etc.

Also, you created a NotificationFilter in filters.py, so use that instead, and make the select like the others where it takes a parameter for filters and passes that to the NotificationFilter.